Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfileController.cs

```diff
@@ -104,25 +104,25 @@ public async Task<ActionResult<IEnumerable<ProfileApiModel>>> SelectProfilesByNa
         /// Action method that handles getting a single user by their email;
         /// GET: /api/profiles/{profileEmail}
         /// </summary>
-        /// <param name="profileEmail">The email of the user being retrieved</param>
+        /// <param name="email">The email of the user being retrieved</param>
         /// <returns>If found, a profile API model version of the profile; if not, it returns a NotFound()</returns>
         [HttpGet("")]
-        [HttpGet("{profileEmail}")]
+        [HttpGet("{email}")]
```

Comment:
the route is defining email as a required, not optional, parameter; regardless of what the method signature suggests. it won't route here if there's empty string for email. if you want it to be optional, define it as `"{email?}"`. if you don't want it to be optional then alter the method from what it is.

Reply:
Made the parameter optional